model-mesh/overlays/odh-model-controller/rbac/role.yaml

```diff
@@ -81,12 +81,26 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
+  - rolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
odh-model-controller needs these because the user can choose to deploy models in any NS on the cluster (as long as modelmesh is enabled on that NS), and we need to create a CRB and a RoleBinding to make the model deployment work as intended

Reply:
Why do models needs access to other namespace though ? Their kinda 'user workload' right ? And you don't give full namespace access to your cluster to those.

Reply:
The models don't need access to other namespaces. The ODH-model controller does. see : 
https://github.com/opendatahub-io/odh-model-controller/blob/main/controllers/inferecenceservice_serviceaccount.go#L46
https://github.com/opendatahub-io/odh-model-controller/blob/main/controllers/monitoring_controller.go#L60

Reply:
If the model don't need access to other namespaces, odh-model-controller does not need to create clusterrolebinding. Whatever installs odh-model-controller (cluster admin, OLM) does need them, but odh-model-controller just need to **have** whatever clusterrolebinding it needs, not to **create** them by itself (which is what we're allowing here).

Reply:
explained and resolved offline with @VannTen and @anishasthana